test/fetchers.test.js

```diff
@@ -358,58 +353,24 @@ describe('testing error page promise resolver', () => {
   });
 });
 
-describe('testing dynamic default resolution', () => {
-  it('Test for simple HTML URLs', () => {
-    assert.equal(getDefault('/foo.html'), '/default.html');
-    assert.equal(getDefault('/foo/bar.html'), '/foo/default.html');
-  });
-
-  it('Test for HTML URLs with selectors', () => {
-    assert.equal(getDefault('/foo.baz.html'), '/default.baz.html');
-    assert.equal(getDefault('/foo/bar.baz.html'), '/foo/default.baz.html');
-  });
-
-  it('Test for HTML URLs with double selectors', () => {
-    assert.equal(getDefault('/foo.baz.zip.html'), '/default.zip.html');
-    assert.equal(getDefault('/foo/bar.baz.zip.html'), '/foo/default.zip.html');
-  });
-
-  it('Test for simple JSON URLs', () => {
-    assert.equal(getDefault('/foo.json'), '/default.json');
-    assert.equal(getDefault('/foo/bar.json'), '/foo/default.json');
-  });
-
-  it('Test for JSON URLs with selectors', () => {
-    assert.equal(getDefault('/foo.baz.json'), '/default.baz.json');
-    assert.equal(getDefault('/foo/bar.baz.json'), '/foo/default.baz.json');
-  });
-});
-
 describe('testing path info resolution', () => {
   const tests = [
     {
-      url: '/',
+      url: '/hä',
       indices: ['index.html', 'readme.html'],
       mount: '',
       expected: [{
-        path: '/index.html',
+        path: '/hä/index.html',
```

Comment:
this is wrong. it should become `/hä.html`.
add also a test for `/hä/` which should become `/hä/index.html`